test/test_all.sh

```diff
@@ -1,41 +0,0 @@
-#!/usr/bin/env sh
-
-# env CMAKE_FLAGS="-DCMAKE_CXX_COMPILER=`which clang++-6.0` -DCMAKE_C_COMPILER=`which clang-6.0`" sh -x test_all.sh
```

Comment:
Could you please specify how to test with sanitizers now ?

Reply:
It is not available now, at least in a form of separate build type. I do it by enabling them in native IDE. If this is a useful functionality (to have a build with enabled sanitizers at cmake configure level), it is probably better to add a separate variables that enable particular sanitizer (as opposed to having separate build type for each sanitizer, since those custom build types may not have everything else configured properly for them.)